aws-athena/views/default-vw_pin_appeal.sql

```diff
@@ -1,7 +1,7 @@
 -- View containing appeals by PIN
 SELECT
     htpar.parid AS pin,
-    pardat.class AS class,
+    REGEXP_REPLACE(pardat.class, '[^[:alnum:]]', '') AS class,
```

Comment:
We need to start manually updating `ccao.class_dict` to ensure that the new classes are included there, otherwise a bunch of existing dbt tests are going to start failing. I opened a separate issue here: https://github.com/ccao-data/data-architecture/issues/362.

Reply:
@wrridgeway The addition of the `OA` classes here is going to cause test failures to appear in our QC workbooks, since those classes aren't yet defined in the `class_dict` table. We should open a quick PR in lieu of #362 to add `class_dict` as a dbt seed and stub out the OA classes. We can do that as a fast-follow or before merging this PR.

Reply:
Addressed in https://github.com/ccao-data/data-architecture/pull/397.